### PR TITLE
Add November update

### PR DESCRIPTION
**Pull request by pablogsal**

### CODE REVIEW

**Review by brettcannon — APPROVED** [review]

File: updates/2022-11-steering-council-update.md [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+  OpenSSL. Unfortunately, some of the bots never came back to life after the
+  restart. Łukasz is investigating.
+- We have been informed that we are lagging a bit with the merging pending accepted Pull Requests.
+  Łukasz is going to help with the merges and the backports. Łukasz also thinks
+  that making auto-merge nicer may help here.
+-  The Steering Council and the developer in residence
+  discussed several things that can be done to improve this situation such as using GitHub Actions
+  or collecting a bunch of search URLs to help people find these PRs.
+- The Steering Council discussed the timeline for the Steering Council Elections. The group asked
+  Ee to prepare to start the process with a similar timeline as previous years: 1
```
Comment (hugovk) [inline updates/2022-11-steering-council-update.md:14]: 2 week nomination period? 
```suggestion
  Ee to prepare to start the process with a similar timeline as previous years: 2
```

File: updates/2022-11-steering-council-update.md [hunk 2]
```diff
@@ -0,0 +1,60 @@
...
+  weeks + 1 week in between + 2 weeks (nominations + discussion + election).
+- The Steering Council discussed some remaining aspects of [PEP 669 – Low Impact
+  Monitoring for CPython](https://peps.python.org/pep-0669/) and decided to accept
+  it.
+- The Steering Council discussed some performance aspects of [PEP 683 – Immortal Objects](https://peps.python.org/pep-0683/).
+- The Steering Council approved (did not veto) the promotion of Filipe Laíns.
```
Comment (hugovk) [inline updates/2022-11-steering-council-update.md:20]: Was Filipe promoted/approved in October?

* https://devguide.python.org/core-developers/developer-log/
* https://discuss.python.org/t/vote-to-promote-filipe-lains/19599/6?u=hugovk

File: updates/2022-11-steering-council-update.md [hunk 3]
```diff
@@ -0,0 +1,60 @@
...
+- The Steering Council discussed [PEP 690 – Lazy Imports](https://peps.python.org/pep-0690/) and decided to reject it. The rejection
+  notice [was posted on Discourse](https://discuss.python.org/t/pep-690-lazy-imports-again/19661/26)
+
+## 2022-11-21
+
+- The Steering Council met with the developer in residence (Łukasz) to discuss progress. Most
+  of Łukasz's time was spent setting up a new machine for bigmem buildbots and
+  Windows debugging/development.
+- The Steering Council discussed the next steps for encouraging mentorship.
+- The Steering Council approved (did not veto) new core developers -- Hugo Kemenade & Kumar Aditya.
```
Comment (hugovk) [inline updates/2022-11-steering-council-update.md:37]:
```suggestion
- The Steering Council approved (did not veto) new core developers -- Hugo van Kemenade & Kumar Aditya.
```

File: updates/2022-11-steering-council-update.md (hunk 2 above)
Comment (pablogsal) [reply updates/2022-11-steering-council-update.md:20]: Yeah, this item is just the confirmation that everything was setted-up up for him

Comment (pablogsal) [inline updates/2022-11-steering-council-update.md:20]:
```suggestion
```

**Pull request merged**